region/maridia/inner-yellow.json

```diff
@@ -1488,7 +1543,10 @@
                   "name": "Base",
                   "notable": false,
                   "requires": [
-                    "canConsecutiveWalljump",
+                    {"or": [
```

Comment:
There isn't a 4->5 anymore.  
The only real difference is that you could shinespark from 4, but that strat was missing already.
`powerbombs` was intended to be placed at the ledge at 4, but placing it in the air would work too.  That concept would open up using wide shots (wave or spazer) to hit the choot from safety and reclimbing.